video/out/x11_common.c

```diff
@@ -377,6 +377,19 @@ static int vo_wm_detect(struct vo *vo)
     return wm;
 }
 
+static void xpresent_set(struct vo_x11_state *x11)
+{
+    int present = x11->opts->x11_present;
+    x11->use_present = x11->present_code &&
+                       ((x11->has_mesa && !x11->has_nvidia && present) ||
+                        present == 2);
+    if (x11->use_present) {
+        MP_VERBOSE(x11, "XPresent statistics enabled.\n");
```

Comment:
are they actually called statistics?

Reply:
Not really. How about `XPresent events` instead?